routes/web.php

```diff
@@ -73,6 +73,8 @@
 
     /** Printing */
     Route::get('/print', [PrintController::class, 'index'])->name('print');
+    Route::post('/print', [PrintController::class, 'noPaper']);
```

Comment:
The URL should resemble its purpose, should be something like `/print/no-paper` or `/print/paper-run-out`. You should also add a name so you can reference it using the `route` global function.